### ISSUE TITLE
[1.17/1.18] Allow Recruiting-List to include Unit-Variations?

### ISSUE THREAD

**Issue by FrostBlade5** [wesnoth/wesnoth#6145]
I'm fascinated by those wonderful unit-variations. https://i.postimg.cc/CxX1WSx5/4.png
I'd be amazing to see those units included to the recruiting-list and the map-editor within a future version of Wesnoth.
I don't see any reason why not to do this, none of those units is op.
Honestly i think that besides from myself there are other people out there thinking the same way.
Can't wait for 1.17 / 1.18 😄

An issue has already been opened regarding the map-editor: [1.17/1.18] Allow Scenario Editor's unit tool to Include Unit Variations. https://github.com/wesnoth/wesnoth/issues/6122

I opened this issue because nowhere in the other thread was mentioned unit-variations should be available not only in the
map-editor but also in the recruiting-list.

If any Developers have an opinion it would be interesting to read it.

**Comment by FrostBlade5** [comment]
Maybe walking corpse and soulless unit-variations fit to the recruting list of the undead race, i wouldn't bother if that gets denied, but really using a faction=Custom race, it litterally says custom, why not be able use unit variations there by using the map editor?

**Comment by Wedge009** [comment]
Unless there's substantially different work involved, I suspect this could be merged with #6122. I assume it would only apply to campaigns played as undead - Descent into Darkness and Secrets of the Ancients (the latter of which apparently already has it? I haven't played 1.15's version of it yet) - as well as MP scenarios playing as undead faction.

**Comment by FrostBlade5** [comment]
> 
> 
> Unless there's substantially different work involved, I suspect this could be merged with #6122. I assume it would only apply to campaigns played as undead - Descent into Darkness and Secrets of the Ancients (the latter of which apparently already has it? I haven't played 1.15's version of it yet) - as well as MP scenarios playing as undead faction.

Agreed, should be merged, no need to have 2 threads for the same issue.

**Comment by CelticMinstrel** [comment]
I think it's fine to consider them as two separate but related issues. It might turn out that they can both be closed at the same time, but they're also distinct requests.

**Comment by knyghtmare** [comment]
There is a modification of the 1.16 add-on server titled "Zombie Variations" which allows such a thing. Perhaps employing that mod and setting it as required?

**Comment by CelticMinstrel** [comment]
I think this is something that would be better integrated into the existing recruit dialogue (possibly as an option) rather than implemented as a modification (as SotA does). But for people wishing to do it in 1.16 in there add-on, that modification sounds like a good option.

**Comment by FrostBlade5** [comment]
!bump

It's been a while.
Any updates on this topic?
Will wesnoth 1.17 or 1.18 include undead unit variations in the map editor or to be selectedable for recruitment?

https://i.postimg.cc/CxX1WSx5/4.png

I was really hoping to see the unit variations being selectable in the editor, it's such a pain editing all the map/scenario files.
